src/ProgressMeter.jl

```diff
@@ -45,6 +45,30 @@ function BarGlyphs(s::AbstractString)
     end
     return BarGlyphs(glyphs...)
 end
+const defaultglyphs = BarGlyphs('|','█', Sys.iswindows() ? '█' : ['▏','▎','▍','▌','▋','▊','▉'],' ','|',)
+
+# Internal struct for holding common properties for progress meters
+Base.@kwdef mutable struct ProgressCore
+    lock::Threads.ReentrantLock = Threads.ReentrantLock()
+    dt::Real                    = Float64(0.1)
```

Comment:
maybe reorder, first the keywords that are meant to be changed by the user, then those who are internal only
`barlen` and `barglyph` are only for `Progress` and should probably go there